apps/aecore/src/aec_governance.erl

```diff
@@ -12,10 +12,13 @@
          locked_coins_holder_account/0,
          minimum_gas_price/1,
          name_preclaim_expiration/0,
-         name_claim_locked_fee/0,
+         name_claim_bid_increment/0,
+         name_claim_fee/1,
+         name_claim_fee_base/0,
          name_claim_max_expiration/0,
          name_protection_period/0,
-         name_claim_preclaim_delta/0,
+         name_claim_preclaim_timeout/0,
```

Comment:
could we not simply keep the name name_claim_preclaim_delta?

Reply:
it is a delta, not really a time

Reply:
@ThomasArts I change that to correlate with name_claim_bid_timout. In both cases it is not a time range that matters but the action that happens when they expire. For such activities, I found word `timeout` more suitable. Similar to timers, which, those functions represent in some abstract form, in fact, configure.